docs/mint.json

```diff
@@ -314,7 +314,7 @@
         {
           "group": "Semantic Search",
           "pages": [
-            "sql/tutorials/rag"
+            "integrations/ai-engines/rag"
```

Comment:
This page should be listed under AI Engines (line 982).

Let's remove it from this line because it is duplicated here. We can create a tutorial and list it here in Use Cases.

Reply:
Yes. Thas why I've added the comment `Note that in the Use Case tutorials, for now, we just link the RAG docs until @martyna-mindsdb adds a new tutorial.`